common/ailments/museum-collection.ts

```diff
@@ -1,16 +1,15 @@
-import Ailment from './ailment'
-import {GameModel} from '../models/game-model'
-import {CardPosModel} from '../models/card-pos-model'
-import {removeAilment} from '../utils/board'
-import {AilmentT} from '../types/game-state'
-
-class MuseumCollectionAilment extends Ailment {
-	constructor() {
+import Ailment from "./ailment"
```

Comment:
Pretty sure the formatting is wrong here

Reply:
how so?